discovery-infra/test_infra/helper_classes/config/nodes_config.py

```diff
@@ -37,14 +22,10 @@ class BaseTerraformConfig(_BaseConfig, ABC):
     libvirt_worker_ips: List[str] = None
```

Comment:
so this variable of `libvirt_worker_ips` shouldn't be under `TFConfig` or the name should be `LibvirtConfig`

Reply:
can't fix everything in one PR,  keep the original code ( If i remember correctly)